addons/website_sale/static/src/js/website_sale.js

```diff
@@ -209,6 +230,18 @@ publicWidget.registry.WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * Sets the url hash from the selected product options.
+     *
+     * @private
+     */
+    _setUrlHash: function ($parent) {
+        var combinations = $parent.find('input.js_variant_change:checked, select.js_variant_change option:selected');
+        combinations = _.map(combinations, function (elem) {
```

Comment:
Use two different variables, especially since the first one should start with a `$`